src/logger/logger.js

```diff
@@ -66,7 +66,7 @@ class CloudWatchLogger {
           handleExceptions: true,
         }
 
-        this._logger.add(new WinstonCloudWatch(cloudwatchConfig))
+        this._logger.add(new WinstonCloudwatch(cloudwatchConfig))
```

Comment:
Can I check if we need to add a `cloudWatchLogs: new AWS.CloudWatchLogs()` in the cloudwatchConfig? Otherwise I'm not sure why we need to add `@aws-sdk/client-cloudwatch-logs` into our package.json.

Reply:
Seems like our logs are still fine!